src/qt/trafficgraphwidget.cpp

```diff
@@ -50,19 +50,28 @@ int TrafficGraphWidget::getGraphRangeMins() const
 void TrafficGraphWidget::paintPath(QPainterPath &path, QQueue<float> &samples)
 {
     int sampleCount = samples.size();
-    if(sampleCount > 0) {
+    if(sampleCount > 0 && fMax > 0) {
         int h = height() - YMARGIN * 2, w = width() - XMARGIN * 2;
         int x = XMARGIN + w;
         path.moveTo(x, YMARGIN + h);
         for(int i = 0; i < sampleCount; ++i) {
             x = XMARGIN + w - w * i / DESIRED_SAMPLES;
-            int y = YMARGIN + h - (int)(h * samples.at(i) / fMax);
+            int y = YMARGIN + h - (int)(h * 1.0 * (fToggle ? (pow(samples.at(i), 0.3) / pow(fMax, 0.3)) : (samples.at(i) / fMax)));
             path.lineTo(x, y);
         }
         path.lineTo(x, YMARGIN + h);
     }
 }
 
+void TrafficGraphWidget::mousePressEvent(QMouseEvent *event)
```

Comment:
This isn't a great UI decision for end users. This functionality is hidden from users. If they discover it, it would be by accident; after having clicked on the Network Traffic widget. 

Let's make this feature more accessible and documented by adding two buttons next to the `reset` button:  one that sets the existing format and another button that enables the new format introduced in this PR.

The `fToggle` variable here should then be made into a persistent setting.

Additionally, when a longer sampling period has been set with the slider; it can take a minute or longer for the format to switch. There are optimizations that need to be done. I have not investigated this.

Reply:
@jarolrod What options are available so far for making GUI changes persistent? Regarding the delay when the slider is moved, I fixed that in a later update.

What more obvious way might there be to reveal the Y axis is toggleable? A tooltip on the graph perhaps? I am reluctant to add extra buttons given the slider is already providing many values in so little space, so I do prefer the clicking on the graph to change it. It may be that someone finds it by accident, but then this could be a pleasant surprise that - an "easter egg" feature!

Reply:
@rebroad 

> What more obvious way might there be to reveal the Y axis is toggle-able?

Yes, button, of course! Checkbox, radio, all those are good.

> A tooltip on the graph perhaps?

Improves from no tooltip, but no,
that's not enough improvement.

> I am reluctant to add extra buttons given the slider is already providing many values in so little space, so I do prefer the clicking on the graph to change it.

What? No, that's an huge slider. An unnecessarily large one.
Sliders can be as small as necessary when they are
combined with a text-box presenting the slided value.
There's a huge empty space below the legend too.
There's plenty of avail space to discard invisible toggles.

> It may be that someone finds it by accident, but then this could be a pleasant surprise that - an "easter egg" feature!

No, an easter egg is meant to be fun or charming in
some way. Hiding actual feature in the same way like
one could do with a true easter egg is transgredes
the principle of least surprise.

Cheers!

Reply:
@jarolrod The latest commit makes the update immediate now (using QWidget::update).